python/tester/upload.py

```diff
@@ -198,6 +220,70 @@ def get_samples_non_lsst(bucket, instrument):
     return result
 
 
+def get_samples_lsst(bucket, instrument):
+    """Return any predefined raw exposures for a LSST instrument.
+
+    Parameters
+    ----------
+    bucket : `S3.Bucket`
+        The bucket in which to search for predefined raws.
+    instrument : `str`
+        The short name of the instrument to sample.
+
+    Returns
+    -------
+    raws : mapping [`str`, mapping [`int`, mapping [`activator.FannedOutVisit`, `s3.ObjectSummary`]]]
+        A mapping from group IDs to a mapping of snap ID. The value of the
+        innermost mapping is the observation metadata for each detector,
+        and a Blob representing the image taken in that detector-snap.
+    """
+    latiss_metadata = {
+        2023082900500: {
+            "ra": 270.14386585558344,
+            "dec": -24.989937309744448,
+            "rot": 360.0,
+            "time": 1693371362.626,
+            "filter": "SDSSr_65mm~empty",
+            "group" : "2023-08-30T04:54:42.780",
+        },
+    }
+
+    # The pre-made raw files are stored with the "unobserved" prefix
+    blobs = bucket.objects.filter(Prefix=f"unobserved/{instrument}/")
+    result = {}
+    for blob in blobs:
+        # Assume that the unobserved bucket uses the same filename scheme as
+        # the observed bucket.
+        m = re.match(LSST_REGEXP, blob.key)
```

Comment:
Indeed we don't have to follow the same convention as in the LSST's raw file bucket, and we could code in information in the path name, organize the files whatever way, etc.  As long as files are loaded in `get_samples_lsst` it should work, and the convention isn't really needed there. We can make that work. But, I think it might be less confusing if all LSST raws follow the same convention and can use utilities in `activator/raw.py` in the same way. Making `dev` as similar to `prod` as possible seems a plus. Because metadata are read from the adjacent json, adding new "unobserved" files is very easy.  We can drop more files (fits and json) into the unobserved folder and then can run the test with more visits right away. 